src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/FileConfigurationProvider.cs

```diff
@@ -88,7 +88,8 @@ private void Load(bool reload)
                     }
                     catch (Exception e)
                     {
-                        HandleException(ExceptionDispatchInfo.Capture(e));
+                        var exception = new InvalidDataException($"Failed to load configuration from file '{file.PhysicalPath}'.", e);
```

Comment:
This will need to be localized properly. The string should be added to [`String.resx`](https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/Resources/Strings.resx) and then referenced here. Are any tests validating this exception type is what should be thrown? I am unfamiliar with this code path but there should be public documentation that identifies expected exceptions.

Reply:
I have done proper localization in both 11d7eb2ad4fa7ac50d68b311542b76732dfb68ba and cc37340265d69462d9bc196742b0bcebdff96c6f.

> Are any tests validating this exception type is what should be thrown?

I haven't found any. Anyway, I can't run tests because I'm currently stuck with a _Could not load PlatformManifest_ error that I'm not able to solve. This is discussed at https://github.com/dotnet/runtime/issues/30501 and https://github.com/dotnet/arcade/issues/3878

> I am unfamiliar with this code path but there should be public documentation that identifies expected exceptions.

I have also documented then new exception in 11d7eb2ad4fa7ac50d68b311542b76732dfb68ba.

Reply:
> I haven't found any. Anyway, I can't run tests because I'm currently stuck with a Could not load PlatformManifest error that I'm not able to solve. This is discussed at #30501 and dotnet/arcade#3878

@0xced have you tried following this link to build and run tests:

https://github.com/dotnet/runtime/tree/master/docs/workflow/building/libraries